### ISSUE TITLE
Logger middleware should implement Hijacker

### ISSUE THREAD

**Issue by samber** [containous/traefik#367]
Hi guys,

I'm having an issue with Websocket. It is probably a regression because it was working 2 or 3 weeks ago.

```toml
# traefik.toml

[entryPoints]
        [entryPoints.http]
                address = ":80"

[file]

[backends]
  [backends.backend1]
      [backends.backend1.servers.server1]
            url = "127.0.0.1:8000"

defaultEntryPoints = ["http"]

[frontends]
  [frontends.frontend1]
    backend = "backend1"
    rule = "Path:/ws/"
```

```sh
# start a websocket server
docker run --rm -p 8000:8000 joshgubler/wscat -l 8000

# establish a connection with a websocket client
wscat -c ws://127.0.0.1/ws/
```

Logs:
```
time="2016-05-12T09:20:16Z" level=warning msg="Unable to hijack the connection: the ResponseWriter doesn't support the Hijacker interface"
172.17.0.1 - - [12/May/2016:09:20:16 +0000] "GET /ws/ HTTP/1.1" 500 21 "" "" 2 "frontend1" "127.0.0.1:8000" 350.13µs
```

I localised the bug between ```v1.0.0-beta.582``` and ```v1.0.0-beta.601```

**Comment by emilevauge** [comment]
Can you confirm that it was working with `v1.0.0-beta.58` and that it is not working since `v1.0.0-beta.601` ?

**Comment by emilevauge** [comment]
I confirm that there is a bug in https://github.com/containous/traefik/blob/master/middlewares/logger.go#L44. It should support Hijacker https://golang.org/pkg/net/http/#Hijacker.
Workaround is to disable access logs for now.
/cc @tooda02

**Comment by emilevauge** [comment]
Fixed in #369

**Comment by samber** [comment]
Nice !! Thanks for your help @emilevauge ;-)
